src/main/java/kr/sesac/aoao/server/user/service/UserServiceImpl.java

```diff
@@ -2,27 +2,71 @@
 
 import static kr.sesac.aoao.server.user.exception.UserErrorCode.*;
 
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Map;
+
+import org.springframework.security.crypto.password.PasswordEncoder;
 import org.springframework.stereotype.Service;
 import org.springframework.transaction.annotation.Transactional;
 
 import kr.sesac.aoao.server.global.exception.ApplicationException;
 import kr.sesac.aoao.server.user.controller.dto.request.SignUpRequest;
+import kr.sesac.aoao.server.user.domain.User;
+import kr.sesac.aoao.server.user.jwt.JwtTokenProvider;
+import kr.sesac.aoao.server.user.repository.UserEntity;
 import kr.sesac.aoao.server.user.repository.UserJpaRepository;
 import lombok.RequiredArgsConstructor;
 
+/**
+ * @since 2024.01.18
+ * @author 이상민
+ */
 @RequiredArgsConstructor
 @Transactional(readOnly = true)
 @Service
 public class UserServiceImpl implements UserService {
 
 	private final UserJpaRepository userRepository;
+	private final JwtTokenProvider jwtTokenProvider;
+	private final PasswordEncoder passwordEncoder;
 
+	/**
+	 * 회원가입
+	 * @since 2024.01.18
+	 * @return User
+	 * @author 이상민
+	 */
 	@Transactional
 	@Override
-	public Long signUp(SignUpRequest signUpRequest) {
+	public User signUp(SignUpRequest signUpRequest) {
```

Comment:
```suggestion
	public SignUpResponse signUp(SignUpRequest signUpRequest) {
```
Service 계층에서 Request 시 DTO 를 받아왔으니 응답도 Response DTO 로 맞추어 주는건 어떨까요??